Fixes #24012 syntax of avc1 codec parameter (#24028)
mdn/content: files/en-us/web/media/formats/codecs_parameter/index.md

  - : Where `oo` is the Object Type Indication value describing the contents of the media more precisely and `A` is the one-digit _audio_ OTI. The possible values for the OTI can be found on the MP4 Registration Authority web site's [Object Types page](https://mp4ra.org/#/object_types). For example, Opus audio in an MP4 file is `mp4a.ad`. For further details, see [MPEG-4 audio](#mpeg-4_audio).
- `mp4v.oo[.V]` (MPEG-4 video)
  - : Here, `oo` is again the OTI describing the contents more precisely, while `V` is the one-digit _video_ OTI.
- `avc1[.PPCCLL]` (AVC video)

>   - : `oo` is the OTI describing the contents, while `PPCCLL` is six hexadecimal digits specifying the profile number (`PP`), constraint set flags (`CC`), and level (`LL`). See [AVC profiles](#avc_profiles) for the possible values of `PP`.

Reviewer: ```suggestion
  - : `PPCCLL` are six hexadecimal digits specifying the profile number (`PP`), constraint set flags (`CC`), and level (`LL`). See [AVC profiles](#avc_profiles) for the possible values of `PP`.
```
